stable_baselines/common/vec_env/vec_normalize.py

```diff
@@ -37,6 +41,30 @@ def __init__(self, venv, training=True, norm_obs=True, norm_reward=True,
         self.norm_reward = norm_reward
         self.old_obs = np.array([])
 
+    def __getstate__(self):
+        """State for pickling. Excludes self.venv, as in general VecEnv's may not be pickleable."""
+        state = self.__dict__.copy()
+        # these attributes are not pickleable
+        del state['venv']
+        del state['class_attributes']
+        # these attributes depend on the above and so we would prefer not to pickle
+        del state['ret']
+        return state
+
+    def __setstate__(self, state):
+        """Restores pickled state. User must call set_venv() after unpickling before using."""
+        self.__dict__.update(state)
+        assert 'venv' not in state
+        self.venv = None
+
+    def set_venv(self, venv):
```

Comment:
Docs. Above docstrings for __get/setstate__ functions could be on their own lines like in rest of the functions